gem_city.rb

```diff
@@ -0,0 +1,38 @@
+# This class represents the town of GemCity
+# This is a town riddled with crime but we can find out how happy the town is
+class GemCity
+  attr_reader :population
+  attr_accessor :officers, :thieves
+
+  def initialize
+    @officers = 1
+    @thieves = 5
+    @population = 50
+  end
+
+  def thieves(thieves_number = @thieves)
+    @thieves = thieves_number
+  end
```

Comment:
Once you use the attr_accessor form, this doesn't do anything new. (It also does two different things, so is probably a bad method.

I would change the test to use theives= so that the attribute handling is consistent.